src/GraphQL/Execution/ErrorInfoProvider.cs

```diff
@@ -56,8 +56,9 @@ public virtual ErrorInfo GetInfo(ExecutionError executionError)
                 var details = _options.ExposeExceptionDetails && _options.ExposeExceptionDetailsMode == ExposeExceptionDetailsMode.Extensions
                     ? executionError.ToString()
                     : null;
+                var userExtensions = executionError.Extensions;
 
-                if (code != null || codes != null || data != null || details != null)
+                if (code != null || codes != null || data != null || details != null || userExtensions != null)
```

Comment:
```suggestion
                if (code != null || codes != null || data != null || details != null || userExtensions?.Count > 0)
```

Add `?.Count > 0` or no?  Either way is fine with me.